### DISCUSSION TITLE
Next Steps

### DISCUSSION THREAD

**Discussion by jaspertey** [lunarstorm/laravel-ddd#60] [General]
I have plans to expand `ddd:model` sometime in the near future and get it to a point of having the same options as `make:model`.

@pelmered, I know you have some stuff going in your fork. Are you able to share what you've done over there, and new challenges if any?

Looking to hear about other real-world use cases for the package and implement into the core.

**Comment by pelmered** [comment]
Yea, I started working a few more generator commands, but I haven't had the time to finish it. The main challenge is limited time. 
What I was working on was:
- Command for generating seeders and migrations inside the domain
- Added options to the model command to generate factory, seeder, migration and policy at the same time. Like you can do in the `make:model` command i Laravel. I always use that and that was something I felt I was missing when I started to use this. 

it is almost done. There was a bug where the migrations was generated in the wrong path, but otherwise I think it is working. 
I can make a draft PR with the changes.
